jenkins/ci.opensuse.org/openstack-diskimage-builder.yaml

```diff
@@ -0,0 +1,96 @@
+- job:
+    name: 'openstack-diskimage-builder'
+    project-type: matrix
+
+    triggers:
+      - timed: '@daily'
+
+    concurent: false
+
+    build-discarder:
+    num-to-keep: 10
+    days-to-keep: -1
+
+    parameters:
+      - bool:
+          name: BOOT_DIB_IMAGE
+          default: false
+          description: Boot the generated image using QEMU/KVM.
+
+    axes:
+      - axis:
+          type: user-defined
+          name: opensuse_release
+          values:
+            - 42.1
+            - 13.1
+      - axis:
+          type: user-defined
+          name: suse_element
+          values:
+              - opensuse
+      - axis:
+          type: slave
+          name: slave
+          values:
+            - cloud-trackupstream
```

Comment:
I'm not sure that we want to run this on our securest worker .. how about using cloud-cleanvm here?